### ISSUE TITLE
error during validation of cf script

### ISSUE THREAD

**Issue by aparhad** [awslabs/aws-cfn-lint-atom#28]
am getting following error when trying to validate cf template:
An unexpected error with cloudformation, cfn-lint, atom-cfn-lint, atom, linter, and/or your playbook, has occurred.
```
Traceback (most recent call last):
  File "/usr/local/bin/cfn-lint", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3138, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3122, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 3151, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 666, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 679, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 867, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'jsonschema~=3.0' distribution was not found and is required by cfn-lint
```

**Comment by aparhad** [comment]
jsonschema 3.2 is installed and available on path /usr/local/bin

**Comment by aparhad** [comment]
Resolved by following these steps:
Created a requirements.txt file using `pip freeze`
Modified the line containing jsonschema = 0.0.0 to 3.2.0
ran `pip install -r requirements.txt`
Sample template with unsupported Type:
```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  # Unsupported Type "AWS::EC2::Instance1"
  MyEC2Instance1:
    Type:                 "AWS::EC2::Instance1"
```
```bash
$ cfn-lint test-lint.yml 
E3001 Invalid or unsupported Type AWS::EC2::Instance1 for resource MyEC2Instance1 in us-east-1
test-lint.yml:5:5
```

**Comment by aparhad** [comment]
Resolved..
